GitUI/CommandsDialogs/RevisionDiffController.cs

```diff
@@ -117,13 +117,12 @@ public bool ShouldShowSubmoduleMenus(ContextMenuSelectionInfo selectionInfo)
 
         public bool ShouldShowMenuEditWorkingDirectoryFile(ContextMenuSelectionInfo selectionInfo)
         {
-            return selectionInfo.IsSingleGitItemSelected && !selectionInfo.IsAnySubmodule && selectionInfo.SingleFileExists;
+            return selectionInfo.IsSingleGitItemSelected && !selectionInfo.IsAnySubmodule && selectionInfo.IsAllFilesExists;
```

Comment:
Does `Edit` support opening multiple files?

Reply:
I don't think so. Plus, there was already the bool "IsSingleGitItemSelected"

Reply:
You are right, `IsSingleGitItemSelected` does the trick.